docs/resources/artifact_property_webhook.md

```diff
@@ -51,7 +54,8 @@ The following arguments are supported:
   * `repo_keys` - (Required) Trigger on this list of repo keys.
   * `include_patterns` - (Optional) Simple comma separated wildcard patterns for repository artifact paths (with no leading slash).\n Ant-style path expressions are supported (*, *\*, ?).\nFor example: "org/apache/**".
   * `exclude_patterns` - (Optional) Simple comma separated wildcard patterns for repository artifact paths (with no leading slash).\n Ant-style path expressions are supported (*, *\*, ?).\nFor example: "org/apache/**".
-* `url` - (Required) Specifies the URL that the Webhook invokes. This will be the URL that Artifactory will send an HTTP POST request to.
-* `secret` - (Optional) Secret authentication token that will be sent to the configured URL
-* `proxy` - (Optional) Proxy key from Artifactory Proxies setting.
-* `custom_http_headers` - (Optional) Custom HTTP headers you wish to use to invoke the Webhook, comprise of key/value pair.
+* `handler` - (Required) At least one is required
+  * `url` - (Required) Specifies the URL that the Webhook invokes. This will be the URL that Artifactory will send an HTTP POST request to.
+  * `secret` - (Optional) Secret authentication token that will be sent to the configured URL
```

Comment:
What is the form of this secret? Exactly what header does it insert

Reply:
The http header is called `x-jfrog-event-auth`

Reply:
So artifactory inserts it's own custom header? I thought it might have been an authorization header. Always a surprise.

@alexhung can we please put that detail in the docs?

Reply:
Not surprised to me since I've been working with the webhook API since last year 😄 

@chb0github Will add the header to the doc 👍🏼